Source/VMware.vSphereDSC/Classes/InventoryBaseDSC.ps1

```diff
@@ -18,83 +18,188 @@ class InventoryBaseDSC : BaseDSC {
     <#
     .DESCRIPTION
 
+    Name of the Inventory Item from the Datacenter we will use.
+    #>
+    [DscProperty(Key)]
+    [string] $Name
+
+    <#
+    .DESCRIPTION
+
+    The full path to the Inventory Item in the Datacenter we will use from the Inventory.
+    The path consists of 0 or more folders.
+    The Root folders of the Datacenter are not part of the path.
+    The parts of the path are separated with "/".
```

Comment:
Folder names in path are separated by "/" "<parent folder name >/<child folder name>"